quiztest/src/main/java/quiztest/main/inventory/VehicleMapper.java

```diff
@@ -0,0 +1,25 @@
+package quiztest.main.inventory;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.springframework.jdbc.core.RowMapper;
+
+public class VehicleMapper implements RowMapper<Vehicle> {
+
+    @Override
+    public Vehicle mapRow(ResultSet rs, int rowNum) throws SQLException {
+        Vehicle vehicle = new Vehicle();
```

Comment:
You can use your pre-existing 7-arg constructor.

Reply:
Thank you!